beacon_node/lighthouse_network/tests/common.rs

```diff
@@ -74,10 +74,11 @@ impl std::ops::DerefMut for Libp2pInstance {
 #[allow(unused)]
 pub fn build_tracing_subscriber(level: &str, enabled: bool) {
     if enabled {
-        tracing_subscriber::fmt()
-            .with_env_filter(EnvFilter::try_new(level).unwrap())
-            .try_init()
-            .unwrap();
+        tracing::subscriber::set_default(
```

Comment:
I _think_ the subscriber is immediately dropped because the return value is not used here. Have you tested this locally?

Reply:
Ah, my bad. I checked that the tests were passing, but didn't realize that no logs were showing. :man_facepalming: